docs/clientdev/overview.rst

```diff
@@ -132,6 +133,20 @@ The resulting POST request would then transmit to::
 .. note:: If any of configuration values are not present, the SDK should notify the user
           immediately that they've misconfigured the SDK.
 
+Event sampling
+--------------
+
+SDKs should allow the user to configure what percentage of events are actually
+sent to the server (the rest should be silently ignored). For example:
+
+.. sourcecode:: python
+
+    sample_rate = options.get('sample_rate') or 1.0
```

Comment:
if `options.get('sample_rate')` returns 0 this expression will evaluate to 1.0 instead, i think theres a legitimate case for setting sample_rate to 0?